cmd/edenPod.go

```diff
@@ -302,6 +305,39 @@ var podPurgeCmd = &cobra.Command{
 					app.Purge = &config.InstanceOpsCmd{Counter: 0}
 				}
 				app.Purge.Counter++
+				volumeConfigs := dev.GetVolumes()
+				for i, oldUUID := range volumeConfigs {
+					v, err := ctrl.GetVolume(oldUUID)
+					if err != nil {
+						log.Fatal(err)
+					}
+					if explicitVolumes {
```

Comment:
Are either all or none of the volumes for an app instance explicit/persistent?

Reply:
If we not define volume option, I purge all, if define, purge only defined. I put comment about rhis into the cobra option comment